src/content/reference/cli.mdx

```diff
@@ -572,6 +572,10 @@ You need to be [logged in](reference/cli.mdx#context) to Okteto before running t
 Only users with administration privileges can destroy a global preview environment.
 If the manifest includes `destroy` commands, they will be executed as part of this command.
 
+| Options                                   |    Type    | Description                                                                                                                                                                                                                                               |
+| :---------------------------------------- | :--------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _--wait_                                  |   (bool)   | Wait until the preview deletion finishes (defaults to true)                                                                                                                                                                                            |
```

Comment:
I think we should remove that part since the default is to not wait if the flag is not available right?

```suggestion
| _--wait_                                  |   (bool)   | Wait until the preview deletion finishes                                                                                                                                                                                  |
```

Reply:
@rberrelleza by default it will wait, because otherwise it would be a breaing change since currently it wait, with 3.0 we'll set default to false

Reply:
then how do you tell it not to wait? `--wait=false` ?

Reply:
Yes, `--wait=false` will schedule the destroy.

Reply:
Ah thanks for the explanation!